packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -52,7 +52,7 @@ export class ProfilesCache {
         return this.profileTypeConfigurations;
     }
 
-    public async getProfileInfo(_envTheia = false): Promise<zowe.imperative.ProfileInfo> {
+    public async getProfileInfo(): Promise<zowe.imperative.ProfileInfo> {
```

Comment:
Since this class is exposed in ZE API, we should have a changelog entry for this breaking change.